### DISCUSSION TITLE
Feature Request - Read & leave comments

### DISCUSSION THREAD

**Discussion by hsdavidkim** [community/community#5596] [Mobile] [answered]
Hello, I'm not sure if this feature is already available or not on the mobile app but I'm not able to read the GitHub comments on my code file in a particular commit. I can read the comment and reply to it on the web but not on the mobile app.

**Comment by aconsuegra** [comment, marked as answer]
Hi @hsdavidkim. You are correct, right now the apps are not displaying comments on code that belongs to individual commits, you need to go to the pull request review screen to see those. I'm going to include your feature request to our internal tracking issue.
